tests/store/tracking/test_sqlalchemy_store.py

```diff
@@ -1778,6 +1809,38 @@ def test_log_batch_same_metric_repeated_multiple_reqs(self):
             self.store, run.info.run_id, params=[], metrics=[metric0, metric1], tags=[]
         )
 
+    def test_log_batch_same_metrics_repeated_multiple_reqs(self):
+        run = self._run_factory()
+        metric0 = Metric(key="metric-key", value=1, timestamp=2, step=0)
+        metric1 = Metric(key="metric-key", value=2, timestamp=3, step=0)
+        self.store.log_batch(run.info.run_id, params=[], metrics=[metric0, metric1], tags=[])
+        self._verify_logged(
+            self.store, run.info.run_id, params=[], metrics=[metric0, metric1], tags=[]
+        )
+        self.store.log_batch(run.info.run_id, params=[], metrics=[metric0, metric1], tags=[])
+        self._verify_logged(
+            self.store, run.info.run_id, params=[], metrics=[metric0, metric1], tags=[]
+        )
+
+    def test_log_batch_null_metrics(self):
+        run = self._run_factory()
+
+        tkey = "blahmetric"
+        tval = None
+        metric_1 = entities.Metric(tkey, tval, int(1000 * time.time()), 0)
+
+        tkey = "blahmetric2"
+        tval = None
+        metric_2 = entities.Metric(tkey, tval, int(1000 * time.time()), 0)
+
+        metrics = [metric_1, metric_2]
+
+        warnings.simplefilter("ignore")
+        with self.assertRaises(MlflowException) as exception_context, warnings.catch_warnings():
+            self.store.log_batch(run.info.run_id, metrics=metrics, params=[], tags=[])
+            warnings.resetwarnings()
```

Comment:
```suggestion
        with self.assertRaises(MlflowException) as exception_context:
            self.store.log_batch(run.info.run_id, metrics=metrics, params=[], tags=[])
```

nit: can we remove unnecessary warning capturing?

Reply:
We removed them in https://github.com/mlflow/mlflow/pull/5685